### PR TITLE
Dropdown

### PR DESCRIPTION
**Pull request by WeszeRita**

### CODE REVIEW

File: src/shared/components/dropdown/dropdown.component.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<button class="todo-status {{ statusClass }}" [cdkMenuTriggerFor]="dropdown">{{ buildTranslationKey(status) | translate }}</button>
+
+<ng-template #dropdown >
```
Comment (bodnya29179) [inline src/shared/components/dropdown/dropdown.component.html]:
```suggestion
<ng-template #dropdown>
```

File: src/shared/components/dropdown/dropdown.component.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  <div class="dropdown" cdkMenu>
+    <button
+      *ngFor="let statusValue of statusValues"
+      cdkMenuItemCheckbox
```
Comment (bodnya29179) [inline src/shared/components/dropdown/dropdown.component.html]: what is this checkbox?

File: src/shared/components/dropdown/dropdown.component.scss [hunk 3]
```diff
@@ -0,0 +1,31 @@
+.todo-status {
+  @apply  px-1  w-44;
```
Comment (bodnya29179) [inline src/shared/components/dropdown/dropdown.component.scss]: extra spaces

File: src/shared/components/editable-field/editable-field.component.ts [hunk 4]
```diff
@@ -29,8 +29,10 @@ export class EditableFieldComponent implements ControlValueAccessor {
   value: string;
   previousValue: string;
 
-  private onChange: (value: string) => void = () => {};
-  private onTouched: () => void = () => {};
+  private onChange: (value: string) => void = () => {
+  };
+  private onTouched: () => void = () => {
+  };
```
Comment (bodnya29179) [inline src/shared/components/editable-field/editable-field.component.ts]: alignment

File: src/shared/components/dropdown/dropdown.component.html (hunk 2 above)
Comment (WeszeRita) [reply src/shared/components/dropdown/dropdown.component.html]: I changed to radio

**Review by bodnya29179 — APPROVED** [review]

**Pull request merged**